Source/WebInspectorUI/UserInterface/Models/Canvas.js

```diff
@@ -215,11 +252,26 @@ WI.Canvas = class Canvas extends WI.Object
         return WI.UIString("Canvas %d").format(this._uniqueDisplayNameNumber);
     }
 
-    is2D()
+    get is2D()
```

Comment:
For consistency with the `Recording` class could this be `isCanvas2D` (same for WebGL ones below)?

Reply:
IMO that's a bit redundant since this is a `WI.Canvas` object (e.g. id find it weird to read `canvas.isCanvas2D`)

FWIW the only reason `WI.Recording` uses `isCanvas2D` instead of just `is2D` is because there's technically nothing about `WI.Recording` that's specific to `<canvas>`.  when i first built it many years ago my thinking was that we'd be able to reuse the general purpose data structure for other kinds of recordings (e.g. JS object mutations, DOM tree shenanigans, etc.)